kumascript/macros/Non-standard_Header.ejs

```diff
@@ -12,15 +12,18 @@ var title = mdn.localString({
 });
 
 var description = mdn.localString({
-  "en-US": "This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.",
-  "es":    "Esta característica no es parte de los estándares. No la uses en sitios Web en producción: no funcionará para todos los usuarios. Podrían haber también incompatibilidades considerables entre distintas implementaciones y el comportamiento podría cambiar en el futuro.",
-  "fr":    "Cette fonctionnalité n'est ni standard, ni en voie de standardisation. Ne l'utilisez pas pour des sites accessibles sur le Web : elle ne fonctionnera pas pour tout utilisateur. Il peut également y avoir d'importantes incompatibilités entre les implémentations et son comportement peut être modifié dans le futur.",
-  "ja":    "この機能は標準ではなく、標準化の予定もありません。公開されているウェブサイトには使用しないでください。ユーザーによっては使用できないことがあります。実装ごとに大きな差があることもあり、将来は振る舞いが変わるかもしれません。",
-  "ko":    "이 기능은 비표준이므로 실제 프로덕션에서 사용하지 마세요. 모든 사용자 환경에서 작동하지 않을 수 도 있으며, 미래에 호환성 문제가 생길 수 있습니다.",
-  "ru":    "Эта возможность не является стандартной и стандартизировать её пока никто не собирается. Не используйте её на сайтах, смотрящих во внешний мир: она будет работать не у всех пользователей. Также могут присутствовать большие несовместимости между реализациями и её поведение может в будущем измениться.",
-  "zh-CN": "该特性是非标准的，请尽量不要在生产环境中使用它！"
+  "en-US": "We do not recommend using this on production sites facing the Web, as it will not work for every user.",
```

Comment:
Minor tweaks
```suggestion
  "en-US": "We do not recommend using this feature on production sites facing the web because it will not work for every user.",
```

Reply:
I'm curious why we start writing "the web" lowercase again (we write it as "the Web" on the [About](https://developer.mozilla.org/en-US/about) and [Web APIs](https://developer.mozilla.org/en-US/docs/Web/API) pages, for example).

Reply:
We have a guideline in our [writing style guide](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide#capitalization), so ideally it should be lower case. In the "About" page, we should change the instances to lowercase, of course, except in "MDN Web Docs".